server/android_notification_server.go

```diff
@@ -82,7 +82,7 @@ func (me *AndroidNotificationServer) SendNotification(msg *PushNotification) Pus
 			return NewErrorPushResponse(err.Error())
 		}
 
-		me.logger.Infof("Sending android push notification for device=%v and type=%v", me.AndroidPushSettings.Type, msg.Type)
+		me.logger.Infof("Sending android push notification for device=%v did%v type=%v", me.AndroidPushSettings.Type, msg.DeviceID, msg.Type)
```

Comment:
Not sure if we want to do this, this will add the log in our push proxy giving us every android deviceID and it may not comply with privacy policy.

If this is OK, the same should be done for iOS

Reply:
1. Can the device ID be used to track down the personal information of the person owning the device (i.e. identify who it is)?
2. @icelander what is the use case or problem behind this addition?

Reply:
It can't be use to determine who it is, but I would not recommend exposing it outside of the MM server

Reply:
@enahum Could we log the `AckId`? That's randomly generated for every push request so it couldn't track a single device but could be correlated to log in `notifications.log`